cg/models/flow_cell/flow_cell.py

```diff
@@ -74,6 +74,12 @@ def full_name(self) -> str:
         """Return flow cell full name."""
         return self.path.name
 
+    @property
+    def sequenced_at(self) -> list[str]:
+        """Return the sequencing date for the flow cell."""
+        date_part: str = self.full_name[:6]
+        return datetime.datetime.strptime(date_part, "%y%m%d")
```

Comment:
Not sure how strptime works but it needs to work on both `231023_A00621_0927_BHXXXXXXXX` and `20231103_LH00188_0024_A2XXXXXXXX`

Reply:
Yes, fixed that now 👍  Nice catch